### DISCUSSION TITLE
Custom index.html

### DISCUSSION THREAD

**Discussion by zie1ony** [facebook/docusaurus#8387] [Q&A] [answered]
I have my own `index.html` that is self-contained file. 
Is it possible to serve it as a root and skip docusaurus toolchain just for root (or any other page)?

**Comment by slorber** [comment, marked as answer]
You can replace `src/pages/index.js` by `static/index.html` and Docusaurus will serve your index as the homepage instead (this applies to any other route as well)

This is a bit out of the scope of Docusaurus. In any case you can replace the build output `build/index.html` (or any other file) by your owns, and you are free to merge/override the Docusaurus static build with any other static build the way you want

**Reply by mellson** [reply]
This works great during SSR, but subsequent page changes to the root, for instance, clicking on the site logo, during CSR, the `static/index.html` doesn't get loaded by Docusaurus. Is there a way to make that work?

**Reply by slorber** [reply]
I think this could work: `logo.link = "pathname:///"`

The `pathname://` protocol is something we use to link to external pages but without using the SPA client-side navigation behavior (history push)
